src/argilla_sdk/settings/_question.py

```diff
@@ -219,23 +221,23 @@ def __init__(
             description=description,
             required=required,
             values=values,
-            settings=RatingQuestionSettings(options=_render_values_as_options(values)),
+            settings=RatingQuestionSettings(options=self._render_values_as_options(values)),
         )
 
     @classmethod
     def from_model(cls, model: RatingQuestionModel) -> "RatingQuestion":
-        instance = cls(name=model.name, values=_render_options_as_values(model.settings.options))
+        instance = cls(name=model.name, values=cls._render_options_as_values(model.settings.options))
         instance._model = model
 
         return instance
 
     @property
     def values(self) -> List[int]:
-        return _render_options_as_values(self._model.settings.options)
+        return self._render_options_as_labels(self._model.settings.options)
```

Comment:
The expected value is `list[int]` but the method returns a `list[str]`. This can be confusing to developers later.  Maybe we can review the naming and signature
```python
def _get_option_values_as_list(options: List[Dict[str, Union[str,int]]]) -> List[Union[str,int]]:
   ...

def values(self) -> List[int]:
   return self._get_option_values_as_list(self._model.settings.options)
```